<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotCanvas</class>
 <widget class="QWidget" name="PlotCanvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="12">
      <widget class="CustomPlotZoom" name="canvas" native="true"/>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="ScreenshotButton">
       <property name="toolTip">
        <string>Save current plot in png</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/screenshot.svg</normaloff>:/icons/icons/screenshot.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="ClearButton">
       <property name="toolTip">
        <string>Clear plot</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/reload.svg</normaloff>:/icons/icons/reload.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="UnZoomButton">
       <property name="toolTip">
        <string>Unzoom plot and enable auto scaling</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/unzoom.svg</normaloff>:/icons/icons/unzoom.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="SaveButton">
       <property name="toolTip">
        <string>Save current plot in txt</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/savecanvas.svg</normaloff>:/icons/icons/savecanvas.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QToolButton" name="LogButton">
       <property name="toolTip">
        <string>Plot in log scale</string>
       </property>
       <property name="text">
        <string>LogScale</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/icons/log.svg</normaloff>:/icons/icons/log.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="8" colspan="4">
      <widget class="QLabel" name="countsLabel">
       <property name="text">
        <string>NPS: 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPlotZoom</class>
   <extends>QWidget</extends>
   <header>customplotzoom.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
